drivers/ieee802154/Kconfig.uart_pipe

```diff
@@ -3,7 +3,9 @@
 
 menuconfig IEEE802154_UPIPE
 	bool "UART PIPE fake radio driver support for QEMU"
-	depends on (BOARD_QEMU_X86 || BOARD_QEMU_CORTEX_M3) && NETWORKING
+	default y
+	depends on (BOARD_QEMU_X86 || BOARD_QEMU_CORTEX_M3) && \
```

Comment:
Do we know why this is limited to these qemu boards?  Could this be `QEMU_TARGET` instead?

Reply:
@tbursztyka ^

Anyway, I think that is out of the scope of this PR, can be fixed in a follow-up PR if needed.